src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -231,11 +231,11 @@ export const addConceptToDictionary = (id, dataUrl) => async (dispatch) => {
 };
 
 export const fetchSourceConcepts = async (source, query, index) => {
-  const url = `/orgs/${source}/sources/${source}/concepts/?q=${query}&limit=0&verbose=true`;
+  const url = `/orgs/${source}/sources/${source}/concepts/?q=${query}*&limit=0&verbose=true`;
```

Comment:
How is the above change related to the ticket?

Reply:
I added *
This is to enable partial word search

Reply:
Is that what the ticket is about?

Reply:
The ticket was about making sure that a concept can be searched by name or id. While making sure that the search works properly I noticed an issue which I raised. [https://github.com/OpenConceptLab/ocl_issues/issues/89](https://github.com/OpenConceptLab/ocl_issues/issues/89) and that was the solution suggested.

Reply:
The changes in this pull request should be only about the ticket and nothing more.

Reply:
Yes, the change is necessary for the ticket because it aids the implementation of searching by code.

Reply:
We might need to create another ticket to handle the optimization of search

Reply:
Where is that ticket?

Reply:
Noted. I have created the ticket on the Jira board

Reply:
OCLOMRS-443

Reply:
Can you use the full url for easy navigation?

Reply:
https://issues.openmrs.org/browse/OCLOMRS-443

Reply:
Every ticket should have a description.

Reply:
I have added a description to the ticket

Reply:
How different is that from this? https://issues.openmrs.org/browse/OCLOMRS-410

Reply:
That ticket handles searching for dictionary and concept on the Add CIEL page. While the newly created ticket will handle search on creating concept while mapping.

Reply:
To make it clearer, can you attach a screenshot of the screen that you are referring to, on the ticket?

Reply:
<img width="1440" alt="screen shot 2019-02-14 at 2 01 53 pm" src="https://user-images.githubusercontent.com/20081126/52788631-d846f280-3061-11e9-8e30-0675976b50f0.png">

This one is for https://issues.openmrs.org/browse/OCLOMRS-410

For the new ticket I created it should handle this search
<img width="1278" alt="screen shot 2019-02-14 at 2 02 32 pm" src="https://user-images.githubusercontent.com/20081126/52788761-37a50280-3062-11e9-9b43-d5ac2fa51468.png">

Reply:
Can you read my comment again?

Reply:
Ok I have done that.

Reply:
Can you now remove the ones on the pr?

Reply:
done